contracts/multisig/src/multisig.rs

```diff
@@ -60,12 +58,33 @@ pub trait Multisig:
     /// - the serialized action data
     /// - (number of signers followed by) list of signer addresses.
     #[label("multisig-external-view")]
+    #[allow_multiple_var_args]
     #[view(getPendingActionFullInfo)]
-    fn get_pending_action_full_info(&self) -> MultiValueEncoded<ActionFullInfo<Self::Api>> {
+    fn get_pending_action_full_info(
+        &self,
+        count: OptionalValue<usize>,
+        offset_id: OptionalValue<usize>,
```

Comment:
It's usually better to have `opt_` in front of optional args, makes for cleaner code. Then, you wouldn't need to call it `unwrapped_`.